script_library/cleanup-k8s.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+for line in $(helm ls -a | awk 'NR > 1 {print $1 }'); do
+    helm delete $line --purge;
+done
+
+for NS in openstack ceph nfs libvirt; do
+   helm ls --namespace $NS --short | xargs -r -L1 -P2 helm delete --purge
+done
+
+rm -rf /var/lib/openstack-helm/*
+rm -rf /var/lib/nova/*
+rm -rf /var/lib/libvirt/*
+rm -rf /etc/libvirt/qemu/*
+findmnt --raw | awk '/^\/var\/lib\/kubelet\/pods/ { print $1 }' | xargs -r -L1 -P16 sudo umount -f -l
+
+for pvc in $(kubectl get pvc -n openstack | awk ' NR > 1 { print $1 ; }'); do
+    kubectl delete pvc $pvc -n openstack --all;
+done
+
```

Comment:
This is great - pretty much aligns with what's suggested in the OSH documentation. It might be nice to clean out OSH's files in /tmp. Most are yml. (If doing a non-CAASP deploy you might run into the ceph UUID in a txt file but we don't have to worry about that here. :) This could easily be done in a follow-up change.

Reply:
Great idea! Will do this in a follow up change.